src/assets/specs/command-api.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: "3.0.0"
+info:
+  title: "Rancher Desktop API"
+  version: 0.0.1
+paths:
+  /v0/list-settings:
+    get:
+      operationId: listSettings
+      summary:  List the current preference settings
+      responses:
+        '200':
+          description: |-
+            200 response
+          content:
+            application/json:
+              schema:
+                "$ref" : "#/components/schemas/preferences"
+
+  /v0/set:
+    put:
+      operationId: updateSettings
+      summary:  Updates the specified preference settings
+      requestBody:
+        description: |-
+          JSON block consisting of some or all of the current preferences,
+          with changes applied to any number of settings the backend supports changing this way.
+        content:
+          application/json:
+            schema:
+              "$ref" : "#/components/schemas/preferences"
+        required: true
+      responses:
+        '202':
+          description: |-
+            202 response
```

Comment:
```suggestion
            The settings were accepted.
```

Reply:
Wasn't clear how these fields are used, apart from documentation.

Reply:
Well, yes, but even as documentation having actual English is far superior to just repeating the code two lines up. For one, it saves having to look up what the HTTP status codes mean (or, alternatively, ensures the next person to come through can verify that the chosen status code matches the intended meaning).